bitbots_vision/src/bitbots_vision/vision.py

```diff
@@ -314,7 +312,10 @@ def _configure_vision(self, config, level):
                     config,
                     self._ball_fcnn)
                 # If we don't use YOLO set the conventional goalpost detector.
```

Comment:
```suggestion
                # When using the FCNN, set the conventional goalpost detector.
```
This way the comment does not have to be changed, if we ever use something other than YOLO to find goalposts.